x-pack/test/api_integration/apis/security_solution/saved_objects/notes.ts

```diff
@@ -18,24 +18,24 @@ export default function ({ getService }: FtrProviderContext) {
     after(() => kibanaServer.savedObjects.cleanStandardList());
 
     describe('create a note', () => {
-      it('should return a timelineId, timelineVersion, noteId and version', async () => {
+      it('should return a timelineId, noteId and version', async () => {
```

Comment:
Could you also add a test to create with timelineID as null and make sure it returns appropriate message and status code?